hub/06-separate-core-from-devices.md

```diff
@@ -0,0 +1,381 @@
+# Separate EdgeX Foundry Core from Devices
+
+First, let's go to the `configs` folder and stub out the core and device services:
+
+``` bash
+cd ~/open-horizon-integration/hub/configs
+```
+
+NOTE: this assumes that the git repo you started with was cloned into your home directory (`~/`). Please adjust accordingly.
+
+Then, let's create separate folders for the `core` and `device` services, and stub them out:
+
+``` bash
+mkdir core && cd $_
+hzn dev service new
+cd ..
+mkdir devices && cd $_
+hzn dev service new
+```
+
+This will create a folder structure with the following files and subdirectories:
+
+```
+core
+> horizon
+  > dependencies
+    .gitignore
+  .gitignore
+  hzn.json
+  pattern-all-arches.json
+  pattern.json
+  service.definition.json
+  service.policy.json
+  userinput.json
+```
+
+## Create the `core` Service
+
+Let's first fill out the `core` Service Definition file.  Go to the appropriate folder and edit `service.definition.json`:
+
+``` bash
+cd ../core/horizon
+vi service.definition.json
+```
+
+Let's fill in the appropriate values:
+
+|| Property | Value
+| description | EdgeX Foundry service without devices
+| sharable | singleton
```

Comment:
Do we definitely need this service to be singleton? Same comment for the device service def below,